src/connectors/scenes/FioAddressDetailsConnector.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import { connect } from 'react-redux'
+
+import type { StateProps } from '../../components/scenes/FioAddressDetailsScene'
+import { FioAddressDetailsScene } from '../../components/scenes/FioAddressDetailsScene'
+import type { State } from '../../types/reduxTypes'
+import { formatExpDate } from '../../util/utils'
+
+const mapStateToProps = (state: State) => {
+  const { fioAddress } = state.ui.scenes
+
+  const out: StateProps = {
+    fioAddressName: fioAddress.fioAddressName,
+    expiration: fioAddress.expiration,
+    expirationFormatted: formatExpDate(fioAddress.expiration),
```

Comment:
expirationFormatted will always be a new string and will force a rerender on every redux change. Move the call to formatExpDate to inside the component.

Reply:
ok